solr/core/src/java/org/apache/solr/filestore/DistribFileStore.java

```diff
@@ -192,7 +192,7 @@ private boolean fetchFileFromNodeAndPersist(String fromNode) {
       try {
         GenericSolrRequest request = new GenericSolrRequest(GET, "/node/files" + getMetaPath());
         request.setResponseParser(new InputStreamResponseParser(null));
-        var response = solrClient.requestWithBaseUrl(baseUrl, client -> client.request(request));
+        var response = solrClient.requestWithBaseUrl(baseUrl, request::process).getResponse();
```

Comment:
This is rather nice.  @gerlowskija, I think we can do away with the lambda-less requestWithBaseUrl I was suggesting.  WDYT?

Reply:
ping @gerlowskija